apm-agent-core/src/main/java/co/elastic/apm/agent/logging/LoggingConfiguration.java

```diff
@@ -298,34 +297,7 @@ public void assertValid(Boolean value) {
         .buildWithDefault(LogFormat.PLAIN_TEXT);
 
     public static void init(List<ConfigurationSource> sources, String ephemeralId) {
-        // The initialization of log4j may produce errors if the traced application uses log4j settings (for
-        // example - through file in the classpath or System properties) that configures specific properties for
-        // loading classes by name. Since we shade our usage of log4j, such non-shaded classes may not (and should not)
```

Comment:
We're not shading anymore. Does that lead to problems when users configure log4j with system properties as it also affects the agent's logging setup?

Reply:
With the approach I used here, we simply override any other way to configure loggers in our isolated system. The only way to configure is through our custom `ConfigurationFactory`, so AFAIU, all other options are ignored.
It doesn't mean that there aren't system properties that may affect us, but not ones that I know of related to the general configuration.

Reply:
Having said that, maybe I should restore this false errors that do not really affect us in case some configs come from system properties, WDYT?